package.json

```diff
@@ -572,8 +576,8 @@
                 "when": "launchJsonStatus == Supported && isWorkspaceTrusted"
             },
             {
-                "view": "vscode-edge-devtools-view.targets",
-                "contents": "If you have any questions or want to learn more, check the [docs on GitHub](command:vscode-edge-devtools-view.viewDocumentation)."
+                "view": "vscode-edge-devtools-view.help-links",
+                "contents": "[Documentation](command:vscode-edge-devtools-view.viewDocumentation)\n[File an Issue](https://github.com/microsoft/vscode-edge-devtools/issues/new)"

```

Comment:
nit: wdyt about creating a github issue template for filing an issue. We might be able to pre-populate with some useful info like extension version, etc?

I'm thinking we could start with something basic like asking for repro steps for a bug, etc

Reply:
Yeah that's a great idea! I'll look into making templates on github

Reply:
I just chatted with Rachel, who did the templates for the webhint extension, and she basically said to use the [online editor of GitHub](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository). You can determine if an issue came from the extension or GitHub, but she didn't know if you can automatically include version numbers and such.